plugins/modules/vmware_custom_attribute.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
```

Comment:
How about adding [is_vcenter](https://github.com/ansible-collections/community.vmware/blob/main/plugins/module_utils/vmware.py#L1122) to check whether vCenter because the custom attribute is a feature that vCenter has?

Reply:
Like [this](https://github.com/ansible-collections/community.vmware/pull/1536/commits/cc2221a11b621c52f458268d66c4c99d82664f0d)?

Reply:
Yes, I think it is good!
Could you please also add it to vmware_custom_attribute_manager module?

Reply:
Stupid from me, I should have thought about adding this to both modules myself. It's done.